gfop/get_food_counts.py

```diff
@@ -121,7 +129,11 @@ def get_dataset_food_counts(gnps_network, metadata, filename_col,
     """
     food_counts, filenames = [], []
     gnps_network = pd.read_csv(gnps_network, sep='\t')
-    sample_types = get_sample_types(sample_types)
+    if ref_metadata != 'gfop':
+        delim = ',' if ref_metadata.endswith('.csv') else '\t'
+        sample_types = pd.read_csv(ref_metadata, sep=delim)[['filename', agg_var]].set_index('filename')
+    else:
+        sample_types = get_sample_types(sample_types, level)
```

Comment:
Add below this line: `sample_types = sample_types.squeeze()` to convert `sample_types` to a Series and avoid the MultiIndex.